docs/_static/js/navbar.js

```diff
@@ -1,5 +1,5 @@
 var searchBox = $("#search-input-wrap");
-var TITLE = ['/install/', '/gluon/' , '/api/', '/docs/', '/github/', '/community/', ];
```

Comment:
removing gluon results an empty space on the navbar, and make people harder to find gluon. 

see http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-10449/4/index.html